element_calcium_imaging/imaging_no_curation.py

```diff
@@ -304,10 +306,10 @@ def make(self, key):
 
                 params = (ProcessingTask * ProcessingParamSet & key).fetch1("params")
                 sampling_rate = (
```

Comment:
`sampling_rate`, `fps`, and `ndepths` should be fetched in one query

Reply:
Thanks.  I have combined the fetches for `fps` and `ndepths`.  `sampling_rate` is the variable name for the fetched value of the `fps` attribute.